pages/linux/nmcli-general.md

```diff
@@ -0,0 +1,29 @@
+# nmcli general
+
+> Manage general settings of NetworkManager.
+> This subcommand can also be called with `nmcli g`.
+> More information: <https://networkmanager.dev/docs/api/latest/nmcli.html>.
+
+- Show the general status of NetworkManager:
```

Comment:
```suggestion
- Show a general status of NetworkManager:
```

Reply:
But there isn't `a general status`, there's just `the general status`. There's only one single status, so we should use the definite article `the`.

Reply:
Yes, it single, but does it matter what status it is? For user maybe, but for us no. :)

Reply:
`the` is correct here